### DISCUSSION TITLE
FireFox: "This add-on could not be installed because it appears to be corrupt"

### DISCUSSION THREAD

**Discussion by joisun** [wxt-dev/wxt#995] [Q&A] [answered]
Is there anyone met this notice when try to install with zip ?
![image](https://github.com/user-attachments/assets/12590940-77d1-49ff-b3b3-28fddd53362a)

but everythin work well on Chrome.
here is my repo:
https://github.com/joisun/findjob-bot
and there is the output for firefox:
https://github.com/joisun/findjob-bot/releases/tag/0.0.1

And by the way, what is the `findjob-bot-0.0.1-sources.zip` is after execute `pnpm run zip:firefox` ? does it matter with the notice ?

Thanks for help!

**Comment by aklinker1** [comment]
Nope, never seen that before. Will poke around a bit later today to see if I can figure out why it does this!

**Reply by aiktb** [reply, marked as answer]
> Nope, never seen that before. Will poke around a bit later today to see if I can figure out why it does this!

No need to check, this has nothing to do with wxt, the development mode should load extensions from `about:debugging#/runtime/this-firefox` instead of `about:addons`

**Reply by joisun** [reply]
thanks, that's the reason
